port.sh

```
@@ -0,0 +1,57 @@
+)
+
+echo "Detected OS: $OS"
+echo "Checking port usage..."
+
+for port in "${ports[@]}"; do
+  echo -e "\n==============================="
+  echo "  Checking port $port"
+
+  if [[ "$OS" == "Linux" ]]; then
+    # Check port on Linux
+    local_usage=$(sudo lsof -i :$port)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)